front/lib/production_checks/checks/check_notion_active_workflows.ts

```diff
@@ -63,17 +63,31 @@ async function areTemporalWorkflowsRunning(
       return latest;
     });
 
+    const isRunning = descriptions.every(
+      ({ status: { name } }) => name === "RUNNING"
+    );
+
+    const error = isRunning
+      ? undefined
+      : new Error(
+          "Statuses of workflows are " +
+            descriptions
+              .map(
+                ({ status: { name }, workflowId }) => `${workflowId}: ${name}`
+              )
+              .join(", ")
+        );
+
     return {
-      isRunning: descriptions.every(
-        ({ status: { name } }) => name === "RUNNING"
-      ),
+      isRunning,
       isNotStalled: latests.every(
-        // Check `latest` is less than 1h old.
-        (d) => d && new Date().getTime() - d.getTime() < 60 * 60 * 1000
+        // Check `latest` is less than 2h old.
+        (d) => d && new Date().getTime() - d.getTime() < 2 * 60 * 60 * 1000
```

Comment:
I don't fully understand it. What is this checking ?

Reply:
this is just increasing the stallness delay (cf pr description). 1h is too short imho since they unstall themselves most of the time in the following hour

Reply:
What do we mean by stall ? No activity completed at all?

Reply:
It means that the latest event in the workflow history is more than 1h old (or 2h old now)
So many activities may have completed in the workflow

Reply:
No no I was not sure I understood what it meant. It's really strange that the workflows show no progress at all for more than 1 hour though